oap-server/mqe-grammar/src/main/antlr4/org/apache/skywalking/mqe/rt/grammar/MQEParser.g4

```diff
@@ -33,6 +33,11 @@ expression
     | topN L_PAREN metric COMMA parameter COMMA order R_PAREN  #topNOP
     | relabels L_PAREN expression COMMA label R_PAREN #relablesOP
     | aggregateLabels L_PAREN expression COMMA aggregateLabelsFunc R_PAREN #aggregateLabelsOp
+    | viewAsSeq L_PAREN expressionList R_PAREN #viewAsSeqOp
```

Comment:
@wankai123 Do we need a kind of function for this? I don't think `viewAsSeq` is so special, as generally, it is just a kind of in-memory logic operations.

Reply:
One suggesting group name is `logical_operator`.
Meanwhile, could you rename `function0` and `function1` to `mathematical_operator0`/`1`?

@mrproliu @wankai123 WDYT?

Reply:
make sense, if there are similar expressions, but the logical processing is different, it is should to distinguish by different group names.

Reply:
`mathematical_operator0/1` is more clear

Reply:
@mrproliu Could you update the existing codes as well by following the new proposed names?

Reply:
Updated, please re-check.